src/txdb.h

```diff
@@ -53,24 +53,28 @@ static const int64_t nMaxBlockDBCache = 2;
 uint64_t GetAvailableMemory();
 /** Get the total physical memory */
 uint64_t GetTotalSystemMemory();
+
+struct CacheConfig
```

Comment:
Please add a doccomment for this structure and each member in it.

I'm not asking (yet) that we exhaustively go though and doc-comment everything, but I'd like to see us do so for everything we touch when doing other work.

Reply:
done